app/src/main/java/com/example/sunmusic/screen/main/MainPresenter.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.sunmusic.screen.main
+
+import com.example.sunmusic.data.repository.AlbumRepository
+import com.example.sunmusic.utils.subscribe
+
+class MainPresenter(private val albumRepository: AlbumRepository) : MainContract.Presenter {
+    private var mView: MainContract.View? = null
```

Comment:
```suggestion
    private var view: MainContract.View? = null
```
có thể để lên constructor